indigo_app/views/tasks.py

```diff
@@ -38,7 +38,10 @@ def get(self, request, *args, **kwargs):
         params.update(request.GET)
 
         # initial state
-        if not params.get('state'):
+        if self.kwargs.get('frbr_uri') and not params.get('state'):
```

Comment:
Why would showing a work now show done and cancelled tasks? That seems odd to me. Couldn't it just be a new filter field? The user can always opt into the other task states by adjusting the filters.